cmake/modules/hunter_add_host_subdirectory.cmake

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) 2016 Fabian Winnen
+# All rights reserved.
+
+# Adds a subdirectory with CMakeList.txt as seperate build with default (host) toolchain
+
+# Usage:
+#   hunter_add_host_subdirectory(host)
+#   hunter_add_host_subdirectory(host
+#       BINARY_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/host-build
+#       ENV_VARS CC CXX
+#   )
+
+# Note:
+#   With polly Android toolchain this fails if you do not unset CC and CXX ENV{} variables.
```

Comment:
Can you please print values of `CC`/`CXX` environment variable before `project` command (before loading toolchain), after `project` command and before running `execute_process` with host build command.